Gordon360/Models/ViewModels/RecIM/UpdateSeriesViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Gordon360.Models.CCT;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class UpdateSeriesViewModel
```

Comment:
Perhaps `SeriesUploadViewModel` and then it can be used for updating and creating?

Reply:
Why would you need an ID on this or the other ViewModel at all? If they are only used for creating and updating resources, then you shouldn't ever be dealing with an ID number until after the creation, at which point you should use a different ViewModel as it is no longer an upload.
See my other recommendation for the SeriesController.UpdateSeries(...) method. Your ID should be passed separately from the data that you are trying to upload.